tardis/energy_input/gamma_packet_loop.py

```diff
@@ -289,9 +288,6 @@ def gamma_packet_loop(
                     energy_out[bin_index, time_index] += rest_energy / (
```

Comment:
Instead of the `energy_out`, shall we dump the information into an array which which record the "escaping" packets information. That will help in getting the gamma-ray spectra in a post-processing step?

Reply:
`energy_out` is such an array, but we can record more packet information if you want. I consider this out of scope for this PR though.